### ISSUE TITLE
Error when using `Unsupported("tsvector")` as a type

### ISSUE THREAD

**Issue by jdpowell1** [Quramy/prisma-fabbrica#206]
**Describe the bug**
When running `prisma generate` with a schema with a type of `Unsupported("tsvector)`, fabbrica generation fails 

**To Reproduce**

Use the schema below, run `prisma generate`.

Error in console: `Not found ExampleCreateInput`

**Expected behavior**
That generation succeeds

**Your Schema**

```
generator client {
  provider        = "prisma-client-js"
  output          = "dist/client"
}

generator fabbrica {
  provider    = "prisma-fabbrica"
  noTranspile = true
}

datasource db {
  provider   = "postgresql"
  url        = env("DATABASE_URL")
}

model Example {
  id                 Int                     @unique
  searchString       String
  searchStringVector Unsupported("tsvector")

  @@index([searchStringVector], type: Gin)
}
```

**Environment:**

- Database kind: PostgreSQL 
- Node.js version: 18.18.0
- `@prisma/client` version: 5.4.2
- TypeScript version: 5.2.2

**Comment by Quramy** [comment]
@jdpowell1 

Hi. Thanks for your reporting this issue .

I fixed this problem and released as `v2.0.2`, please checkout this version.

Note:

I made prisma-fabbrica skip to generate factory of the `Example` model, which has unsupported type field.
Because prisma-client don't allow to call `.create` or `.update` methods of this model.

> Note: If a model has mandatory Unsupported fields, the generated client will not include create or update methods for that model.

https://www.prisma.io/docs/concepts/components/prisma-schema/data-model#unsupported-types

**Comment by jdpowell1** [comment]
Yes, that's perfect, thanks for the quick fix!
